com.unity.hlod/Editor/HLODTreeNodeRenderer.cs

```diff
@@ -22,7 +22,7 @@ public static HLODTreeNodeRenderer Instance
             }
         }
         #endregion
-        
+        private Vector3[] m_allocedVertices = new Vector3[8];
```

Comment:
Sorry about keeping to notify trivial points!
It seems like an another typo! (m_allocedVertices -> m_allocatedVertices), and I think it will be probably the last one!

Reply:
Fixed.